axelrod/tests/integration/test_tournament.py

```diff
@@ -34,9 +33,9 @@ def test_full_tournament(self):
         tournament = axelrod.Tournament(name='test', players=strategies,
                                         game=self.game, turns=2,
                                         repetitions=2)
-        tmp_file = tempfile.NamedTemporaryFile()
+        filename = "test_outputs/test_tournament.csv"
```

Comment:
The directory should exist, I've put a `README.md` file in there so cloning the library should put it there right?